source/client/process_impl.cc

```diff
@@ -270,10 +270,31 @@ ProcessImpl::mergeWorkerStatistics(const std::vector<ClientWorkerPtr>& workers)
   return merged_statistics;
 }
 
+void ProcessImpl::allowApiV2(envoy::config::bootstrap::v3::Bootstrap& bootstrap) {
+  auto* admin_layer = bootstrap.mutable_layered_runtime()->add_layers();
+  admin_layer->set_name("admin layer");
+  admin_layer->mutable_admin_layer();
+  envoy::config::bootstrap::v3::RuntimeLayer* runtime_layer =
+      bootstrap.mutable_layered_runtime()->add_layers();
+  runtime_layer->set_name("static_layer");
+  Envoy::ProtobufWkt::Value proto_true;
+  proto_true.set_string_value(std::string("true"));
+  (*runtime_layer->mutable_static_layer()
+        ->mutable_fields())[std::string("envoy.reloadable_features.enable_deprecated_v2_api")] =
```

Comment:
why does this string need to be wrapped in a std::string here?

In general, is there a way to simplify this line, or break it into multiple lines?  After some reading i've determined that mutable_fields is likely a map of sorts, and envoy.reloadable_features.enable_deprecated_v2_api is a key in it that we need to set to a boolean proto (proto_true), but the line right now is a little unwieldy.

Reply:
There was some code in Envoy that inspired that, but wrapping in std::string indeed isn't needed. Removed!